st2api/st2api/controllers/v1/aliasexecution.py

```diff
@@ -82,6 +82,9 @@ def post(self, payload):
             'source_channel': payload.source_channel
         }
 
+        if hasattr(payload, 'notification_route'):
+            context['route'] = getattr(payload, 'notification_route')
```

Comment:
It would also be nice to update docs for that (we already have a small section which describes ChatOps specific context attributes).

Besides that, LGTM